Formula/v/veryfasttree.rb

```diff
@@ -0,0 +1,82 @@
+class Veryfasttree < Formula
+  desc "Efficient phylogenetic tree inference for massive taxonomic datasets"
+  homepage "https://github.com/citiususc/veryfasttree"
+  url "https://github.com/citiususc/veryfasttree/archive/refs/tags/v4.0.4.tar.gz"
+  sha256 "27c779164f4fa0c75897a6e95b35f820a2a10e7c244b8923c575e0ea46f15f6b"
+  license "GPL-3.0-only"
+  head "https://github.com/citiususc/veryfasttree.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "libomp"
+
+  fails_with :gcc do
+    version "4" # fails with GCC 4.x and earlier
+    cause "Requires gcc >=5"
+  end
+
+  def install
+    system "cmake", "-B", "build", *std_cmake_args
```

Comment:
Let's follow the style used by other formulae:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```
Also, we should really do this:
```suggestion
    system "cmake", "-S", ".", "-B", "build", "-DUSE_SHARED=OFF", "-DUSE_NATIVE=OFF", *std_cmake_args
```
- `USE_SHARED` follows: https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae
- We don't want `USE_NATIVE` because the binaries we build should be portable.

Reply:
USE_SHARED=ON was created to publish Veryfasttree on Debian without third-party libraries, with the exception of bxzstr and CLI11. bxzstr was excluded because it is not available in the Debian repositories, and CLI11 was excluded because it is a modified version with extended support for certain parameters, allowing Veryfasttree to imitate the FastTree2 CLI syntax.

we can use USE_SHARED=OFF or USE_SHARED=ON with "boost-align, boost-core, boost-sort, bzip2, zlib, robin-map" as dependencies.

This is the debian source repository: https://salsa.debian.org/med-team/veryfasttree

Reply:
> USE_SHARED=ON with "boost-align, boost-core, boost-sort, bzip2, zlib, robin-map" as dependencies.

Let's do this, please.

You'll probably need
```ruby
depends_on "robin-map" => :build
depends_on "boost"
uses_from_macos "bzip2"
uses_from_macos "zlib"
```

Reply:
Sorry, just realised my initial suggestion had a typo. I meant
```
USE_SHARED=ON
```
from the beginning.